README.md

```diff
@@ -20,4 +20,26 @@ We are constantly working to improve this project, help us with your donation as
 
 **Donate Bitcoin**
 + **Payment Address:** `bc1q7sgzww3l0fk3dyy9mzcjxdw9rla3yvhuyf0qku`  
-![QR Code](https://user-images.githubusercontent.com/56504893/116758583-a0031300-aa25-11eb-9624-0009346d2290.png)
\ No newline at end of file
+![QR Code](https://user-images.githubusercontent.com/56504893/116758583-a0031300-aa25-11eb-9624-0009346d2290.png)
+
+# Merge and Release Process
+
+## Branches in use
+
+### Development
+
+PRs should be made against and merged into the [`dev-5.x`](https://github.com/hossein-zare/react-native-dropdown-picker) branch, which is set as the `default` branch on github.
+
+### Release
+
+Releases are currently made from teh [`5.x`](https://github.com/hossein-zare/react-native-dropdown-picker/tree/5.x) branch.
```

Comment:
There's a typo "teh" :)

Reply:
:facepalm: and that's why review never ever hurts :-). Great spot